setup.py

```diff
@@ -14,7 +14,7 @@
         'numpy',
         'pandas',
         'petl',
-        'plotly >= 2, < 3',
+        'plotly >= 3, < 4',
```

Comment:
woah, taking the leap! Do we need to make sure everyone's on the same page about this? presumably people currently have plotly 2.x installed and expect to use that in their jupyter notebooks.